README.md

```diff
@@ -108,6 +120,32 @@ dp.dimensions.Add(dim);
 MetricSender.sendMetric(dp);
 ```
 
+Sending custom metric in the Web API Controller layer on down for `ASP.Net Core Web API with Lambda` implementations is curtailed by the fact that the lambda context object is not available. However, the SignalFx C# Lambda Wrapper provides extension method `SignalFx.LambdaWrapper.Extensions.WrapperExtensions.AddMetricDataPoint()` for type `Microsoft.AspNetCore.Http.HttpResponse` for exporting metric datapoints to SignalFx. Below is an example of sending metrics from within a Web API Controller method.
+
+```cs
+...
+using SignalFx.LambdaWrapper.Extensions;
+...
+[HttpGet]
+public IEnumerable<string> Get()
+{
+    ...
+    DataPoint dataPoint = new DataPoint()
```

Comment:
are the parenthesis needed here in `new DataPoint()` ?

Reply:
Apparently not!